spec/support/m2bf2_xquery_helpers.rb

```diff
@@ -0,0 +1,41 @@
+# helper methods for rspec
```

Comment:
I'm wondering if the name of this file is misleading because it currently has xquery in it.  xquery is bf1 converter, xslt for bf2 converter ...